lib/client-assets.php

```diff
@@ -134,6 +134,13 @@ function gutenberg_register_scripts_and_styles() {
 		filemtime( gutenberg_dir_path() . 'build/is-shallow-equal/index.js' ),
 		true
 	);
+	wp_register_script(
+		'wp-token-list',
+		gutenberg_url( 'build/token-list/index.js' ),
+		array(),
```

Comment:
Should we add `lodash` here

Reply:
Yes. 😅

Reply:
Observation: Too much human time spent toward considering adding / removing dependencies from various places (`client-assets.php`, `package.json`). Could really do for some automation / build testing.

Reply:
Yes, I raised it once during Core JS meeting. We agreed we should do it, but there is no action in the area. Ideally, we could offer Webpack plugin which would generate JSON file to consume by WordPress. To make it work we would have to ensure that all deps are moved to WordPress packages though.

Reply:
Do you have a link handy to where it was discussed previously? Is it worth adding again to the agenda?